package com.syarah.yousuf.task.core.errorhandler;

@kotlin.Metadata(mv = {1, 9, 0}, k = 1, xi = 48, d1 = {"\u0000\u001e\n\u0002\u0018\u0002\n\u0002\u0010\u0000\n\u0000\n\u0002\u0018\u0002\n\u0002\b\u0002\n\u0002\u0018\u0002\n\u0000\n\u0002\u0010\u0003\n\u0000\b\u0007\u0018\u00002\u00020\u0001B\u000f\b\u0007\u0012\u0006\u0010\u0002\u001a\u00020\u0003\u00a2\u0006\u0002\u0010\u0004J\u000e\u0010\u0005\u001a\u00020\u00062\u0006\u0010\u0007\u001a\u00020\bR\u000e\u0010\u0002\u001a\u00020\u0003X\u0082\u0004\u00a2\u0006\u0002\n\u0000\u00a8\u0006\t"}, d2 = {"Lcom/syarah/yousuf/task/core/errorhandler/NetworkErrorMapper;", "", "json", "Lkotlinx/serialization/json/Json;", "(Lkotlinx/serialization/json/Json;)V", "map", "Lcom/syarah/yousuf/task/core/errorhandler/exception/RemoteException;", "throwable", "", "error-handler_developmentDebug"})
public final class NetworkErrorMapper {
    @org.jetbrains.annotations.NotNull()
    private final kotlinx.serialization.json.Json json = null;
    
    @javax.inject.Inject()
    public NetworkErrorMapper(@org.jetbrains.annotations.NotNull()
    kotlinx.serialization.json.Json json) {
        super();
    }
    
    @org.jetbrains.annotations.NotNull()
    public final com.syarah.yousuf.task.core.errorhandler.exception.RemoteException map(@org.jetbrains.annotations.NotNull()
    java.lang.Throwable throwable) {
        return null;
    }
}